helm/charts/hpe-csi-driver/templates/hpe-csi-controller.yaml

```diff
@@ -185,7 +186,8 @@ spec:
               value: unix:///var/lib/csi/sockets/pluginproxy/csi-extensions.sock
             - name: LOG_LEVEL
               value: {{ .Values.logLevel }}
-          imagePullPolicy: {{ .Values.imagePullPolicy | quote }}
```

Comment:
Lets keep the pull policy the same from values.yaml

Reply:
Done